app/scripts/controllers/preferences.js

```diff
@@ -112,6 +112,10 @@ export default class PreferencesController {
       ///: END:ONLY_INCLUDE_IF
       useExternalNameSources: true,
       useTransactionSimulations: true,
+      // Turning OFF basic functionality toggle means turning ON this disableExternalServices flag.
+      // Whenever disableExternalServices is true, certain features will be disabled.
+      // The flag is false by Default, meaning the toggle is ON by default.
+      disableExternalServices: false,
```

Comment:
Nit: Maybe better to call this something like `useExternalServices` or `enableExternalServices, to avoid the double-negative.